openmls/src/group/core_group/mod.rs

```diff
@@ -306,6 +314,7 @@ impl CoreGroup {
     // struct {
     //     PreSharedKeyID psk;
     // } PreSharedKey;
+    #[cfg(any(feature = "test-utils", test))]
     pub fn create_presharedkey_proposal(
```

Comment:
Some of these are exposed for tests, others are not. I wonder why and how we could oganise these better. Maybe moving the `pub` `test` functions to a separate `impl` block?

Reply:
Yes, I guess that could work. Though in this case it's marked as test-only because the feature is not yet fully implemented. (see #141).